### ISSUE TITLE
[a11y] Shared image is not labeled in Android Sharesheet

### ISSUE THREAD

**Issue by hoangduong1607** [flutter/flutter#77078]
I am using the [Share plugin](https://pub.dev/packages/share) to share image. There is an accessibility issue that the image preview is not labeled in the Android Sharesheet.

## Steps to reproduce
- Have an image saved locally.
- Use [`Share.shareFiles()`](https://pub.dev/documentation/share/latest/share/Share/shareFiles.html) to share the image.

## Observation
![image](https://user-images.githubusercontent.com/28851958/109697786-3bf8d580-7bc1-11eb-8b49-a95d971bf6bb.png)

## Device models:  
Pixel 3, Pixel 4 XL

**Comment by stuartmorgan** [comment]
Closing, as share is now deprecated in favor of [share_plus](https://pub.dev/packages/share_plus).
